source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -883,7 +883,7 @@ GLSSharpNavierStokesSolver<dim>::postprocess_fd(bool firstIter)
 {
   if (this->simulation_control->is_output_iteration())
     {
-      this->write_output_results(this->present_solution);
+      this->write_output_results(this->present_solution, firstIter);
```

Comment:
You can get from the simulation control if this is the first iteration, since the function already uses the simulation control, why not use it directly?
See : https://github.com/lethe-cfd/lethe/blob/master/include/core/simulation_control.h#L186

Reply:
Oh with the `is_at_start()`? I did not know, good suggestion I will change that

Reply:
in the end I used `get_step_number() == 0` because `is_at_start()` is true for both iteration `0` and `1`

Reply:
Makes sense